cli/src/lib/pub.ts

```diff
@@ -162,17 +163,47 @@ const pub = (options: PublishOptions) => {
 
   const pubOpts = parsePublishOptions(options)
 
-  if (options.stdin) {
+  const handleFileRead = async () => {
+    const filePath = processPath(options.fileRead!)
+    if (!filePath) {
+      signale.error('File path is required when reading from file.')
+      process.exit(1)
+    }
+
+    try {
+      signale.pending('Reading file...')
+      let bufferData = readFile(filePath)
+      if (bufferData.length >= 256 * 1024 * 1024) {
+        signale.error('File size exceeds 256MB, not supported')
```

Comment:
```suggestion
        signale.error('File size over 256MB not supported by MQTT.')
```

Isn't it better to let users know why we cannot handle data larger than 256MB?